src/pluto/operation/bilinear_resize.cr

```diff
@@ -1,9 +1,9 @@
 module Pluto::Operation::BilinearResize
-  def bilinear_resize(width : Int32, height : Int32) : Image
+  def bilinear_resize(width : Int32, height : Int32) : self
```

Comment:
This and below changes allow `image.bilinear_resize(...)` and `grey_image.bilinear_resize(...)` to return their respective types (with `image` being an instance of `Image` and `grey_image` being an instance of `GreyImage`).